tests/Fields/FieldTransformerTest.php

```diff
@@ -64,4 +64,128 @@ public function it_normalizes_required_validation()
             $this->configToVue(['required' => true, 'validate' => 'min:3|required|email'])['validate']
         );
     }
+
+    /** @test */
+    public function it_removes_redundent_config_options()
+    {
+        $fromVue = FieldTransformer::fromVue([
+            'fieldtype' => 'text',
+            'handle' => 'test',
+            'type' => 'inline',
+            'config' => [
+                'input_type' => 'text', // Fieldtype default.
+                'icon' => 'text',  // Fieldtype default.
+                'character_limit' => 100, // This one has been changed.
+                'instructions_position' => 'above', // Not fieldtype related, should be kept.
```

Comment:
I would think these sorts of keys _should_ be filtered out. The default value is `above` so there's no need to save it.

Reply:
Ah, good point. Didn't even think of the "standard" config options w/ defaults. I'll get them filtered out too.